website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -347,6 +351,8 @@ A `default_node_pool` block supports the following:
 
 ~> At this time there's a bug in the AKS API where Tags for a Node Pool are not stored in the correct case - you [may wish to use Terraform's `ignore_changes` functionality to ignore changes to the casing](https://www.terraform.io/docs/configuration/resources.html#ignore_changes) until this is fixed in the AKS API.
 
+* `ultra_ssd_enabled` - (Optional) Used to specify whether the UltraSSD is enabled. See [the documentation](https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks) for more information.
```

Comment:
What does this mean: On the control plane nodes? On the default node pool? Are they enabled by default?

My guess is this is "can Ultra SSD Disks be used in the Default Node Pool?" but that should be clarified here